src/voice/Playback.ts

```diff
@@ -49,7 +49,7 @@ export class Playback extends EventEmitter implements IDestroyable {
     constructor(private buf: ArrayBuffer, seedWaveform = DEFAULT_WAVEFORM) {
         super();
         this.context = new AudioContext();
-        this.resampledWaveform = arrayFastResample(seedWaveform, PLAYBACK_WAVEFORM_SAMPLES);
+        this.resampledWaveform = arrayFastResample(seedWaveform ?? DEFAULT_WAVEFORM, PLAYBACK_WAVEFORM_SAMPLES);
```

Comment:
Shouldn't this be covered by the default value for the function parameter above...?

Reply:
Should, unless a null squeezes in here